src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/CanonicalXmlNodeList.cs

```diff
@@ -38,7 +38,7 @@ public override int Count
         public int Add(object value)
         {
             if (!(value is XmlNode))
-                throw new ArgumentException(SecurityResources.GetResourceString("Cryptography_Xml_IncorrectObjectType"), "node");
+                throw new ArgumentException(SR.GetResourceString("Cryptography_Xml_IncorrectObjectType"), "node");
```

Comment:
The strongly typed thing is built based on the resx.  It's an improvement we made in corefx over how we build in netfx :smile:.